docs/dev/framework/models/_index.md

```diff
@@ -96,6 +98,31 @@ $pages = PageModel::findBy('pid', 1, $options);
 $pages = PageModel::findByPid(1, $options);
 ```
 
+If the model has relations of type `hasOne` or `belongsTo` defined in the [DCA](/framework/dca), setting `'eager' => true`
+will make Contao load all related records of those types within the same database call, using joined query under the hood.
+Columns from each joined table will be prefixed with the column identifier from the main table, followed by double underscore
+character. This allows you to further filter your query with `having` option, for example:
+
+```php
+// tl_article.php
+'author' => array
+(
+    'foreignKey'              => 'tl_user.name',
+    'relation'                => array('type'=>'hasOne', 'load'=>'lazy')
+)
```

Comment:
```suggestion
'author' => [
    'foreignKey' => 'tl_user.name',
    'relation' => ['type'=>'hasOne', 'load'=>'lazy'],
],
```

Reply:
I actually like your version more; that fragment was just copy-pasted from the core😁

Reply:
Yes, the core still uses a legacy style.